recipe/meta.yaml

```
@@ -131,6 +131,8 @@ outputs:
         - python
```

Needed for `pip check` test

```suggestion
        - python
        - pip
```